internal-services/catalog/pulp-push-disk-images-task.yaml

```diff
@@ -152,7 +152,9 @@ spec:
                 FILE=$(jq -c --arg i "$i" '.staged.files[$i|tonumber]' <<< "$COMPONENT")
                 SOURCE=$(jq -er '.source' <<< "$FILE")
                 FILENAME=$(jq -er '.filename' <<< "$FILE")
-                gzip -d "${SOURCE}.gz" || echo "didn't find mapped file: ${SOURCE}.gz"
+                if [ -f "${SOURCE}.gz" ] ; then
```

Comment:
I agree this is nicer. But just to be clear, the behavior stayed the same here, right? (unless the gz file is corrupted or something)

Reply:
Mentioned in other comment but I only just now realized that the qcow2s are not zipped

Reply:
Yeah, ok. Would mind adding a comment above this?

Reply:
Comment added, thanks.